internal/cluster/transition_test.go

```diff
@@ -4242,12 +4307,13 @@ var _ = Describe("Ocs Operator use-cases", func() {
 			}
 
 			Expect(db.Create(&cluster).Error).ShouldNot(HaveOccurred())
+			mockIsValidMasterCandidate()
 			for i := range t.hosts {
 				t.hosts[i].ClusterID = clusterId
 				Expect(db.Create(&t.hosts[i]).Error).ShouldNot(HaveOccurred())
 			}
 			cluster = getCluster(clusterId, db)
-			if t.dstState == models.ClusterStatusInsufficient {
+			if t.dstState == models.ClusterStatusInsufficient || t.name == "ocs enabled, 3 nodes, with role of one as auto-assign" {
```

Comment:
Why do we need to add the extra condition here?

Reply:
Done